### PR TITLE
[EM-300] Add see details report to pr size metric

### PR DESCRIPTION
**Pull request by femartinez**
## What does this PR do?

- Add "See Details" button to the "PR Size" metric chart.
- Creates a new service and view for listing the pull requests that compounds the metric chart.

Resolves #XX

[EM-300 Implement Details Report for PR Size metric](https://rootstrap.atlassian.net/jira/software/projects/EM/boards/42?selectedIssue=EM-300)

### CODE REVIEW

**Review by brunoparma88 — APPROVED** [review]

File: app/views/pull_requests/pull_request_size_prs_repository/index.erb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<%= render 'development_metrics/development_metrics_sidebar', enabled_users_section: @enabled_users_section %>
+  <div class="metrics-container">
+    <div class="row title_sidebar">
+        <div class="col-md-6">
+          <b class="rute_title_grey"><%= params[:repository_name] +'/'%></b>
```
Comment (adlamas) [inline app/views/pull_requests/pull_request_size_prs_repository/index.erb:5]: Why is the param variable in the view? It's not better to filter it on the controller? 🤔

File: spec/requests/pull_requests/pull_request_size_prs_repository/index_spec.rb [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+    it 'renders pull requests index view' do
+      expect(response).to render_template(:index)
+    end
+
+    it 'returns pull requests variable with data' do
+      expect(assigns(:pull_requests)).not_to be_empty
+    end
+
+    it 'returns ony one PR from each range' do
```
Comment (adlamas) [inline spec/requests/pull_requests/pull_request_size_prs_repository/index_spec.rb:82]:
```suggestion
    it 'returns only one PR from each range' do
```

File: app/views/pull_requests/pull_request_size_prs_repository/index.erb (hunk 1 above)
Comment (femartinez) [reply app/views/pull_requests/pull_request_size_prs_repository/index.erb]: Fix with repository name validation [commit e8ae885d811055f77f917b04d981b2ae730e7d72](https://github.com/rootstrap/rs-code-review-metrics/pull/527/commits/e8ae885d811055f77f917b04d981b2ae730e7d72)

File: app/controllers/pull_requests/pull_request_size_prs_repository_controller.rb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+module PullRequests
+  class PullRequestSizePrsRepositoryController < PullRequestsController
+    def repository
```
[the remark this answers is not in the document]
Comment (femartinez) [reply app/controllers/pull_requests/pull_request_size_prs_repository_controller.rb:3]: That method is called by the parent class (PullRequestsController), in the index action. 
Looking that code, is something similar to a polymorphism in controllers.

**Review by adlamas — APPROVED** [review]

**Pull request merged**